CMakeLists.txt

```diff
@@ -45,6 +58,11 @@ set(CMAKE_MACOSX_RPATH ON)
 # Export all symbols
 SET(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 
+# # Allow for the distribution directory to be explicitly specified
```

Comment:
Why is this code commented out? (We generally avoid checking in commented-out code unless it's an example of what not to do or debugging aid)

Reply:
Fixed by removing the block. This was no longer needed once everything got moved to use `install` and CPack for the `distrib` target.